content/quiz_events.md

```diff
@@ -4,21 +4,134 @@
+    - It is not required to be in question and answer form, except in the case of a reference question.
+    - The quizzer’s answer must include all unique words included in both the question and the answer.
+
+Mispronounced names are still recognizable as the answer.
+
+Only the first response of the quizzer will be considered, however, if the response by the quizzer is in context and is not incorrect, the quizzer will be allowed their full 30 seconds to give the additional information necessary to satisfy the question and the answer.  Example: INT Who has been making it grow? A. God.  If the quizzer said “Apollos” they have given incorrect information (even though the name “Apollos” is in context) and would be called incorrect.  If they had said “I planted the seed”, “Apollos watered it”, they have not given any incorrect information, are in context and will be given the 30 seconds to supply the correct answer required. 
 
 ### Incorrect
+The quizzer's response is incorrect when:
+- Incorrect information has been given.
+    - e.g If the wrong person (i.e. Father, Son, Holy Spirit) in the deity (God) is given.
+- The information given within the time limit is incomplete.
+- It does not fulfill all requirements specific to the Question Type.
+     - e.g. Any word is omitted or added in a Finish-The-Verse, Finish-This, or Quote Question or the wrong verse is begun.
 
 ### Challenges
+Challenges exist so quizzers can contest rulings made by quiz officials.
+
+The team captain or co-captain may challenge if:
+- There is reason to believe the quiz officials have applied the rule book incorrectly
+- There has been a mispronunciation by the quizmaster
+- There has been an invalid question
+- The next question has not been called
+    - A challenge must be made at the time of the quizmaster's decision and before the next question is called. If a time-out is called immediately after a ruling, and a quizzer stands to challenge, the challenge may be done prior to the time-out.
+- No conferring has occurred between the quizzer challenging and anyone else. Conferring with anyone else prior to a challenge will result in:
+    - The challenge being disallowed
+    - A foul being awarded to the quizzer.
+
+Once a team has requested an explanation of a ruling, and that explanation has been given, all teams lose the right to challenge.
+- If an explanation of a ruling is requested, the quizmaster should pause before giving the explanation, to allow all teams the opportunity to challenge.
+
+The result of a challenge may not be challenged.
+
+If the challenge is accepted, points or error points that were awarded for the initial ruling of the judge will be deducted.  Points or error points which reflect the new ruling of the judge will be recorded.  A new question of the same number will be given if the question is ruled illegal.
```

Comment:
I don't think is necessary. Thoughts?

Reply:
Entirely agreed.

Reply:
I think the result of a challenge may not be challenged line is useful. The rest isn't necessary but it's mildly useful for clarity. However, even better might be something like if a challenges is accepted, the resulting ruling change's effects will be taken into account, or something like that.